ui/src/shared/components/LineGraph.tsx

```diff
@@ -63,20 +70,18 @@ class LineGraph extends PureComponent<LineGraphProps, State> {
 
   public async componentDidMount() {
     this.isComponentMounted = true
-    const {data} = this.props
-    await this.parseTimeSeries(data)
+    const {data, fluxData} = this.props
+    await this.parseTimeSeries(data, fluxData)
   }
 
   public componentWillUnmount() {
     this.isComponentMounted = false
   }
 
-  public async parseTimeSeries(data) {
-    const {location} = this.props
+  public async parseTimeSeries(data, fluxData) {
+    const timeSeries = await this.convertToDygraphData(data, fluxData)
 
-    const timeSeries = await timeSeriesToDygraph(data, location.pathname)
-    const innerTimeSeries = _.get(timeSeries, 'timeSeries', [])
-    this.isValidData = await manager.validateDygraphData(innerTimeSeries)
+    this.isValidData = await manager.validateDygraphData(timeSeries.timeSeries)
```

Comment:
is it possible that this fails?

Reply:
yep, im wrapping it with the time series to dygraph try catch